### DISCUSSION TITLE
Ability to set and retain the column widths of a O2M field in Table Layout

### DISCUSSION THREAD

**Discussion by MarkBurvs** [directus/directus#22601] [Feature Requests]
### Summary

In the Directus console, when a Collection contains a O2M field with its Layout option under the Interface section set to "Table" view, it's often desirable to drag and resize some of the column widths to give better visibility of the field contents when viewing an Item in that Collection. However, on doing so, those column widths are not retained when the user leaves the page and comes back again. 

### Motivation

The current behaviour results in the user having to resize the columns every time they return to an Item view, which is a huge pain and greatly limits the user experience of the Table view for these field types. 
It also runs counter to the behaviour of the resizeable columns when viewing any collection in Table view, as these DO retain the column widths set by the user when returning to the page.

![1  After setting column widths](https://github.com/directus/directus/assets/9461289/11f647d9-b682-428e-b658-7c4b5beabfad)
![2  On returning again to the same page](https://github.com/directus/directus/assets/9461289/17c4cb56-e9dd-4491-9595-36b7e323b771)

### Detailed Design

This could just be as simple as having Directus remember the last column widths set by a user whenever they are manually changed.

### Must Have:
- A Columns widths in O2M fields in Table Layout should be stored when adjusted by the user, and re-applied when the user visits an Item page in the Collection.

### Should Have:
- C

### Could Have:
- D A way of specifying what the column widths should be by default when creating the O2M field.

### Won't Have:
- E

### Drawbacks

-

### Alternatives

- 

### Adoption Strategy

-

**Comment by TomazicM** [comment]
Any chance of implementation? Absolutely needed!
